modules/claims_api/app/controllers/concerns/claims_api/v2/claims_requests/tracked_items.rb

```diff
@@ -9,7 +9,11 @@ module TrackedItems
         def find_tracked_items!(claim_id)
           return if claim_id.blank?
 
-          local_bgs_service.find_tracked_items(claim_id)[:dvlpmt_items] || []
+          @tracked_items ||= local_bgs_service.find_tracked_items(claim_id)[:dvlpmt_items] || []
```

Comment:
Is this memoization necessary? We only call `find_tracked_items` [once](https://github.com/department-of-veterans-affairs/vets-api/blob/6e02f9c43c19bddf05232024c65939c5ceb3a85e/modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb#L389) in the controller.

Reply:
It will be called more than once in the tracked_item module

Reply:
If that's the case, should we pass in `tracked_items` and do the memoization at the controller level? I think concerns should be as stateless as possible to encourage reusability, and this instance variable could cause future confusion.